app/run.js

```diff
@@ -12,64 +12,64 @@ const shouldQuit = app.makeSingleInstance((cliargs, cwd) => {
 })
 
 if (shouldQuit) {
-  return app.quit()
-}
+  app.quit()
```

Comment:
I know, this is not ideal :) I also like to bail out early if possible. Also helps with readability with less scope. We could add a `// eslint-disable-line` here as well and call it a day.